scripts/setup_linux.sh

```diff
@@ -216,10 +216,15 @@ conda activate garage
   fi
 
   # Fix Box2D install
+  # box2d-py 2.3.4 works on both Linux & OSX and is
+  # actively mantained by OpenAI.
+  # Before that, box2d-py works only on Linux,
+  # and Box2D-kengz works only on OSX.
   # See https://github.com/openai/gym/issues/100
   # See https://github.com/pybox2d/pybox2d/issues/82
+  # See https://github.com/openai/gym/pull/1120 (fixed)
```

Comment:
do we still need the step where we remove the other Box2D/box2d packages?

We previously did this because pip found various Box2D packages in the dependency tree and installed them all.

Reply:
Yeah I agree, they are not necessary. It's a fresh installation in a new conda environment.